### PR TITLE
Update Dependency & Add Hint

### PR DESCRIPTION
**Pull request by taooceros**
Update Dependency to Flow.Launcher.Plugin to 1.4.0

Change the empty query action to change the query text to specific shortcut so that user can type argument. (If you think it is not as good as simply run the command, I can revert it).
Add hint when typing the shortcut.

### CODE REVIEW

File: Wox.Plugin.Runner/Wox.Plugin.Runner.csproj [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <OutputPath>..\Output\Debug\Wox.Plugin.Runner</OutputPath>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Flow.Launcher.Plugin" Version="1.1.0" />
+    <PackageReference Include="Flow.Launcher.Plugin" Version="1.4.0" />
     <PackageReference Include="System.Data.DataSetExtensions" Version="4.5.0" />
```
Comment (jjw24) [inline Wox.Plugin.Runner/Wox.Plugin.Runner.csproj:22]: I don't think the version is correct. Think we are on 2.1.0 currently

File: Wox.Plugin.Runner/Runner.cs [hunk 2]
```diff
@@ -1,134 +1,162 @@
...
+        private static List<Result> FuzzySearchCommand(string shortcut, string[] terms)
+        {
+            return RunnerConfiguration.Commands.Select(c => new Result()
+            {
+                Score = Context.API.FuzzySearch(shortcut, c.Shortcut).Score,
+                Title = c.Shortcut,
+                SubTitle = c.Description,
+                Action = _ =>
+                {
+                    Context.API.ChangeQuery($"{Context.CurrentPluginMetadata.ActionKeyword} {c.Shortcut} {string.Join(' ', terms)}");
```
Comment (jjw24) [inline Wox.Plugin.Runner/Runner.cs:86]: This is a better implementation

File: Wox.Plugin.Runner/Wox.Plugin.Runner.csproj (hunk 1 above)
Comment (taooceros) [reply Wox.Plugin.Runner/Wox.Plugin.Runner.csproj:22]: emm I am not sure what you means. This is the plugin dependency version from flow, and the latest is 1.4.0.
 I will bump the version later.

Comment (jjw24) [reply Wox.Plugin.Runner/Wox.Plugin.Runner.csproj:22]: My bad, I misread it.

**Review by jjw24 — APPROVED** [review]

**Comment by jjw24** [comment]
I am afk, do you want to build a release?

**Comment by taooceros** [comment]
> I am afk, do you want to build a release?

Sure

**Comment by taooceros** [comment]
@jjw24 I think I don't have the write access toward this repository, so maybe you may still need to create a release for that. 😂

**Comment by jjw24** [comment]
Just sent an invite

**Pull request merged**